src/Security/RememberLoginHash.php

```diff
@@ -102,7 +102,7 @@ public static function getLogoutAcrossDevices(): bool
         if (is_bool(static::$logoutAcrossDevices)) {
             return static::$logoutAcrossDevices;
         }
-        return static::config()->get('logout_across_devices');
+        return (bool)static::config()->get('logout_across_devices');
```

Comment:
I assume this one is because the return type is explicit, so this protects against `null`?
Is this ever hit before cache is generated?

Reply:
I haven't come across that in this case, but I feel it's kinda irrelevant. Good practice and clean, stable code is the objective here.

Reply:
Please don't add code unrelated to the issue being fixed, it's just a distraction that slows downs the pull-request process.

Any sort of attempt to globally enforce type hinting should be handled in an different issue